readme.md

```diff
@@ -6,11 +6,19 @@
 npm install --save-dev eslint-config-jquery
 ```
 
-Configure ESLint with a `.eslintrc` file using the following contents:
-```json
-{
-	"extends": "jquery"
-}
+Version 4.0.0 and above work for ESLint version 9.0.0 and above. Use version 3.0.2 for older ESLint versions.
+
+In your eslint.config file, import it and add it to your flat config array.
+
+```javascript
+import jquery from "eslint-config-jquery";
+
+export default [
+    // ... any other configurations,
+    jquery,
+    // ... any other configurations
+];
```

Comment:
@timmywil that's probably a good idea; the way we use the config is too "picky"; we should perhaps just apply it on a global level and then provide overrides in separate blocks.